blocks/hero-horizontal-tabs/hero-horizontal-tabs.js

```diff
@@ -31,14 +31,22 @@ export default function decorate(block) {
   const image = getImage(block);
   const text = getText(block);
   const tabs = createTabs(block, text);
-
+  if (tabs) {
   // move the tab riders in front
-  const wrapper = block.parentElement;
-  const container = wrapper.parentElement;
-  container.insertBefore(wrapper, container.firstElementChild);
-
-  addTabs(tabs, block);
+    const wrapper = block.parentElement;
+    const container = wrapper.parentElement;
+    container.insertBefore(wrapper, container.firstElementChild);
 
+    addTabs(tabs, block);
+  } else {
+    block.firstElementChild.remove();
+    const divs = document.querySelectorAll('div'); // Select all div elements
+    divs.forEach((div) => {
+      if (div.textContent.trim() === 'Contents') {
```

Comment:
I think we should support lowercase as well, maybe `.toLowerCase()` the textcontent and check against the lowercase string

Reply:
Is `Contents` used here as a keyword without requiring localization?

Reply:
Contents is the name of the row in the table of hero-horizontal-block while authoring as shown below. 
So apart from adding the null check for the tabs, the row names (i.e. Image and Contents) were getting appended to the html and hence it was required to remove them in case of null tab.
<img width="636" alt="Screenshot 2023-11-01 at 11 11 11 PM" src="https://github.com/hlxsites/sunstar/assets/45959816/8434b0b8-041a-4874-891f-e8f12839e9b9">